### DISCUSSION TITLE
FW_Restart

### DISCUSSION THREAD

**Discussion by skull-man** [nlef/moonraker-telegram-bot#235] [Show and tell]
Hello, I've improved the bot a bit, now it can do fw_restart. Maybe it will be interesting for you?

**Comment by skull-man** [comment]
[main.txt](https://github.com/nlef/moonraker-telegram-bot/files/10302118/main.txt)
[websocket_helper.txt](https://github.com/nlef/moonraker-telegram-bot/files/10302119/websocket_helper.txt)

**Comment by nlef** [comment]
@skull-man 
Thank you for your interest in the project and expanding its functionality.
Your suggestion has already been implemented in the branch 'development'. Please take a look at https://github.com/nlef/moonraker-telegram-bot/commit/46ce9e3b8b88f1a0f47cc8bb8119118c3ebb15db

**Reply by skull-man** [reply]
@nlef maybe even honestly I can be useful, if anything, please contact!  thanks for your work
